qemu/tests/virtio_console.py

```diff
@@ -280,7 +280,7 @@ def test_rw_host_offline_big_data():
         guest_worker.cmd("virt.clean_port('%s'),1024" % port.name, 10)
         match, tmp = guest_worker._cmd("virt.send('%s', (1024**3)*3, True, "
                                        "is_static=True)" % port.name, 30)
-        if match is None:
+        if match is not None:
```

Comment:
@xiangchunfu In this case, we expect match is None, which means the communication should fail, because this case is trying to transfer data on offline mode. If the match is not None, no matter 0/1, the communication is successful, but the case fails. Thanks.